test/typings/test-d/case.test-d.ts

```diff
@@ -184,4 +196,8 @@ function testCaseValue(eb: ExpressionBuilder<Database, 'person'>) {
   expectError(
     eb.case('gender').whenRef('first_name', '=', 'last_name').then('Mr.').end(),
   )
+
+  expectError(
+    eb.case('gender').when('male').then('Mr.').elseRef('no_such_column'),
+  )
```

Comment:
This is not the right test case (location). Let's move it to the end of the first test case.